### PR TITLE
UF-4935: Add integration tests

### PR DESCRIPTION
**Pull request by ronnberg**

### CODE REVIEW

**Comment by foobar-1337** [comment]
Don't forget to update properties-repo as well.

File: src/test/java/se/sundsvall/eventlog/integration/db/SchemaVerificationTest.java [hunk 1]
```diff
@@ -12,14 +14,15 @@
 import java.nio.file.Paths;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace.NONE;
 
 @SpringBootTest
```
Comment (foobar-1337) [inline src/test/java/se/sundsvall/eventlog/integration/db/SchemaVerificationTest.java:19]: `@DataJpaTest` ?

Comment (ronnberg) [reply src/test/java/se/sundsvall/eventlog/integration/db/SchemaVerificationTest.java]: yes

File: src/main/java/se/sundsvall/eventlog/api/model/Metadata.java [hunk 2]
```diff
@@ -12,7 +12,7 @@ public class Metadata {
 	@NotBlank
 	private String key;
 
-	@Schema(description = "The value", example = "john123")
+	@Schema(description = "The value", example = "jakarta.")
```
[the remark this answers is not in the document]
Comment (ronnberg) [reply src/main/java/se/sundsvall/eventlog/api/model/Metadata.java]: oops

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**